qiskit_metal/toolbox_metal/layer_stack_handler.py

```diff
@@ -0,0 +1,347 @@
+                    fill_value = False
+                else:
+                    self.logger.warning(
+                        f'The \"fill\" value is neither True nor False.'
+                        f'You have:{value}.  '
+                        f'Will return NULL for fill value.')
+                props['thickness'] = thickness
+                props['z_coord'] = z_coord
+                props['material'] = material
+                props['fill'] = fill_value
+            except Exception as ex:
+                self._warning_search_minus_chip(layer_number, datatype, ex)
+        result = list()
+        for item in properties:
+            result.append(props[item])
+        return tuple(result)
+
+    def is_layer_unique(self) -> bool:
+        """Check to sort on layer number make sure they are unique.
+        This method is not so relevant, since layers can have datatype entries.
+        Thus there can be rows with same layer number.
+        """
+        return self.ls_df['layer'].is_unique
+
+    def _read_csv_df(self, abs_path: str) -> None:
+        #ASSUME that self.filename_csv_df is valid file path and name.
+        #https://best-excel-tutorial.com/59-tips-and-tricks/485-how-to-display-a-single-quote-in-a-cell
+        try:
+            self.ls_df = pd.read_csv(abs_path)
+
+        except BaseException as error:
+            self.logger.warning(
+                f'Not able to create pandas dataframe.'
+                f'File:{abs_path} not imported. Expected a CSV formatted file.')
+            self.logger.error(f'The exception: {error}')
+
+    def get_fill_for_chip_layer_datatype(self,
+                                         chip_name: str,
+                                         layer_num: int,
+                                         datatype=0) -> Tuple[bool, bool]:
+        """Determine if geometry should be filled in Ansys.
+
+        Args:
+            chip_name (str): Name of chip which has the layer of interest.
+            layer_num (int): The unique layer number through out all chips.
+                            The same layer number can  not be used on multiple chips.
+            datatype (int, optional):  Datatype of the layer number. Defaults to 0.
+
+        Returns:
+            Tuple[bool, bool]: 1st bool denotes if value for fill was available,
+                            2nd value denotes value for fill. Default is None.
+        """
+        found_fill_value = False
+        fill_value = None
+        mask = (self.ls_df['layer']
+                == layer_num) & (self.ls_df['datatype'] == datatype) & (
+                    self.ls_df['chip_name'].str.contains(chip_name))
+        search_result_df = self.ls_df[mask]
+        if len(search_result_df) > 0:
+            try:
+                value = search_result_df.fill.iloc[0].strip('\'')
+                if value in TRUE_STR:
+                    found_fill_value = True
+                    fill_value = True
```

Comment:
I imagine want the actual value, not True here?

Reply:
The expected value for fill is either True or False.  I just allow user to enter strings within the TRUE_STR or FALSE_STR.

Reply:
I think this addresses the question and will close.  We can re-open if needed.